src/lib/Libutil/avltree.c

```diff
@@ -219,8 +219,17 @@ freenode(node *n)
 static int
 compkey(rectype *r1, rectype *r2)
 {
-	int n = ix_keylength ? memcmp(r1->key, r2->key, ix_keylength) : strcmp(r1->key, r2->key);
-	if (n || ix_dupkeys == AVL_NO_DUP_KEYS)
+	int n;
+	if (ix_keylength)
+		n = memcmp(r1->key, r2->key, ix_keylength);
+	else {
```

Comment:
we can remove the braces. upto you.

Reply:
i would keep it - makes more readable i feel